README.md

```diff
@@ -746,7 +759,12 @@ Ph.D. Thesis, ETH-Zurich, Zurich, Switzerland, 2017. [PDF](https://www.research-
 *[Event-based Vision for High-Speed Robotics](http://rpg.ifi.uzh.ch/docs/PhD17_Mueggler.pdf)*,  
 Ph.D. Thesis, University of Zurich, Zurich, Switzerland, 2017.
 - See also [Theses from Delbruck's group at INI](https://www.ini.uzh.ch/~tobi/wiki/doku.php?id=publications#phd_thesis)
-
+- <a name="Matolin10PhD"></a>Matolin, D.,
+*[Asynchroner CMOS-Bildsensor mit erweitertem Dynamikbereich und Unterdrueckung zeitlich redundanter Daten](http://docplayer.org/docview/27/9947799/#file=/storage/27/9947799/9947799.pdf)*, 
+Ph.D. Thesis, Technische Universitaet Dresden & AIT, 2010.
+- <a name="Nowakowska11MSc"></a>Nowakowska, A.,
+*[Recognition of a vision approach for fall detection using a biologically inspired dynamic stereo vision sensor](http://repositum.tuwien.ac.at/obvutwhs/download/pdf/1623418?originalFilename=true)*,
```

Comment:
Reformat entry.
Start a new sub-section for master theses.

Reply:
Ok